src/actions/moon-phase.js

```diff
@@ -1,69 +1,46 @@
 const Axios = require('axios');
-const Types = require('../action-types/moon-phase');
+const { makeActionCreator } = require('utils/redux-helpers');
+const { MOON_PHASE: Types } = require('action-types');
 
 const actions = exports;
 
-// Marks the moment we start waiting for the moon phase
-actions.beginLoad = () => {
+actions.load = makeActionCreator(
+    Types.MOON_LOAD,
+    {}, // No schema needed, as no args are passed
+    {
+        async: () => {
 
-    return {
-        type: Types.MOON_LOAD_BEGIN
-    };
-};
+            // We'll want to lookup the moon phase for this very millisecond
+            const now = Date.now();
 
-// Marks the moment we've received some good info about the moon phase
-actions.loaded = (moon) => {
+            return new Promise((resolve, reject) => {
```

Comment:
I don't think this needs to be explicitly wrapped in a promise.

Reply:
I wrapped this one because we're mutating the result before it goes to the reducers, so we can control when this action resolves

Reply:
If you are holding onto a promise (which you are, via `Axios.get()`) then you don't need to create one explicitly.  Returning `getMoonPhase` at the bottom of the promise body also isn't doing anything, I don't think.  This "passthrough" stuff seems leftover from thunk-land.